source/common/http/conn_manager_impl.cc

```diff
@@ -1346,15 +1346,15 @@ void ConnectionManagerImpl::ActiveStream::refreshCachedTracingCustomTags() {
   if (!connection_manager_.config_.tracingConfig()) {
     return;
   }
-  Tracing::CustomTagMap& customTagMap = getOrMakeTracingCustomTagMap();
+  Tracing::CustomTagMap& custom_tag_map = getOrMakeTracingCustomTagMap();
```

Comment:
In many cases there will be no custom tags. Can we avoid creating the map until we know there are custom tags on the route or HCM level below?

Reply:
Ok I did like it before IIRC :), I'll add it back.
PS, checking the tracing config before all even though there are custom tags on the route level is needed, the tracing will be processed only when tracing in HCM configured.

Reply:
Fixed